code/modules/power/pipecleaners.dm

```diff
@@ -203,15 +202,17 @@ By design, d1 is the smallest direction and d2 is the highest
 	full_w_class = WEIGHT_CLASS_SMALL
 	grind_results = list("copper" = 2) //2 copper per pipe_cleaner in the coil
 	usesound = 'sound/items/deconstruct.ogg'
+	/// Currently set cable color
+	var/pipe_cleaner_color = "red"
 
 /obj/item/stack/pipe_cleaner_coil/cyborg
 	is_cyborg = 1
 	custom_materials = null
 	cost = 1
 
 /obj/item/stack/pipe_cleaner_coil/cyborg/attack_self(mob/user)
-	var/pipe_cleaner_color = input(user,"Pick a pipe cleaner color.","Cable Color") in sortList(list("red","yellow","green","blue","pink","orange","cyan","white"))
-	pipe_cleaner_color = pipe_cleaner_color
+	var/selected_color = input(user, "Pick a pipe cleaner color.", "Cable Color") in sortList(list("red", "yellow", "green", "blue", "pink", "orange", "cyan", "white"))
```

Comment:
```suggestion
	var/selected_color = input(user, "Pick a pipe cleaner color.", "Cable Color") in list("red", "yellow", "green", "blue", "pink", "orange", "cyan", "white")
```

Reply:
adding a `null` provision for if the person chooses to cancel the event would be nice as well